`TypeError: Cannot read property 'tick' of undefined` when not using `-p`

Using `asciidoc-link-check@1.0.13`:

```
/tmp/node_modules/asciidoc-link-check/index.js:39
                bar.tick();
                    ^

TypeError: Cannot read property 'tick' of undefined
    at /tmp/node_modules/asciidoc-link-check/index.js:39:21
    at /tmp/node_modules/async/dist/async.js:243:13
    at replenish (/tmp/node_modules/async/dist/async.js:440:21)
    at iterateeCallback (/tmp/node_modules/async/dist/async.js:424:21)
    at /tmp/node_modules/async/dist/async.js:321:20
    at /tmp/node_modules/async/dist/async.js:245:17
    at /tmp/node_modules/asciidoc-link-check/index.js:49:13
    at Request._callback (/tmp/node_modules/link-check/lib/proto/http.js:37:17)
    at Request.self.callback (/tmp/node_modules/request/request.js:185:22)
    at Request.emit (events.js:315:20)
make: *** [Makefile:9: check-links] Error 123
```

which ultimately fails the run.

This is a problem because it means you have to enable progress when running CI where it messes up the update.

--- a/index.js
+++ b/index.js
@@ -36,7 +36,9 @@ module.exports = function asciidocLinkCheck(asciidoc, opts, callback) {
                 linkCheckResult.link = link;
                 linkCheckResult.statusCode = 0;
                 linkCheckResult.status = 'ignored';
-                bar.tick();
+                if (opts.showProgressBar) {
+                    bar.tick();
+                }
                 callback(null, linkCheckResult);
                 return;
             }

